bleachbit/CLI.py

```diff
@@ -63,7 +63,7 @@ def cleaners_list():
     list(register_cleaners())
     for key in sorted(backends):
         c_id = backends[key].get_id()
-        for (o_id, o_name) in backends[key].get_options():
+        for (o_id, _o_name) in backends[key].get_options():
```

Comment:
@monosans 

Thank you: The `_o_name` style is better than the `_` style because it avoids current and future conflicts with gettext, and in case the variable is useful later, it's clear what the variable contains.